plugins/base/src/main/kotlin/renderers/html/HtmlRenderer.kt

```diff
@@ -92,6 +101,7 @@ open class HtmlRenderer(
                 childrenCallback()
             }
             node.dci.kind in setOf(ContentKind.Symbol) -> div("symbol $additionalClasses") {
+                node.extra.extraHtmlAttributes().forEach { attributes[it.extraKey] = it.extraValue }
```

Comment:
Does it fix some bug or is it a corner case? Why do we need to pass extra attributes in 7 specific cases only? What happens if I forget to do it, or how do I know if I need to do the same thing for other types of content/blocks?